src/state/slices/opportunitiesSlice/resolvers/thorchainsavers/index.ts

```diff
@@ -0,0 +1,25 @@
+import type { ThornodePoolResponse } from '@shapeshiftoss/swapper'
+import axios from 'axios'
+import { getConfig } from 'config'
+import { DefiProvider } from 'features/defi/contexts/DefiManagerProvider/DefiCommon'
+import { bnOrZero } from 'lib/bignumber/bignumber'
+
+import type { GetOpportunityIdsOutput } from '../../types'
+
+export const thorchainSaversOpportunityIdsResolver = async (): Promise<{
+  data: GetOpportunityIdsOutput
+}> => {
+  const { data: opportunitiesData } = await axios.get<ThornodePoolResponse[]>(
+    `${getConfig().REACT_APP_THORCHAIN_NODE_URL}/lcd/thorchain/pools`,
+  )
+
+  if (!opportunitiesData) return { data: [] }
+
+  const opportunityIds = opportunitiesData
+    .filter(opportunity => bnOrZero(opportunity.savers_depth).gt(0))
+    .map(opportunity => `${DefiProvider.ThorchainSavers}::${opportunity.asset}`)
```

Comment:
The pool ID / AssetId only would technically work for now, however, that's hacky, since another opportunity could technically use the same PoolId/AssetId in the future.

Very unlikely, but might as well, if anything for debugging purposes. We can always deserialize it when needed

Reply:
Discussed this one out, let's use IDs. Unless there's another native asset staking opportunity we integrate (e.g ETH2 staking), there will never be any collision so let's not over-engineer this and make this diff a monster